app/helpers/invitations_helper.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module InvitationsHelper
+  def inviter_name(resource)
+    User.where(id: resource.invited_by_id).first.presence.try(:name) ||
+      'Someone'
```

Comment:
This should be a key in our translation files!

Reply:
Could you clarify what you're thinking exactly? I think I'm a little confused.

Reply:
Oh sorry! What I meant to say was someone should be a value in en.yml and that needs to be translated to our other languages too.